test/whitequark/test_numbered_args_after_27_0.parse-tree-whitequark.exp

```diff
@@ -0,0 +1,6 @@
+s(:numblock,
+  s(:send, nil, :m),
+  s(:numparams, 9),
+  s(:send,
+    s(:lvar, :_1), :+,
+    s(:lvar, :_9)))
```

Comment:
ooc since i've not made a big parser change like this before, did you use a script to import these tests? Can you comment what the command line was for posterity's sake if so?

Reply:
Those are the tests imported by the `import_whitequark` script. As explained here I ran the script then applied the changes to make them pass and finally split them into smaller PRs easier to review/merge.